src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -269,6 +269,8 @@ public function dump(array $options = [])
             foreach ($this->generateProxyClasses() as $c) {
                 $code .= $c;
             }
+
+            $code .= sprintf("\nreturn new %s();\n", $options['class']);
```

Comment:
this should be removed (moved to the dumpContainer method)

Reply:
why. like that, it's consistent with the "asFiles=true" mode!

Reply:
because there are ppl out there that use the component standalone an don't want a container to be created and destroyed immediately :)

Reply:
Nice! So this is a bug fix !

Reply:
Fixed